administration/intro.adoc

```diff
@@ -78,50 +62,24 @@ $ kubectl create configmap -n kubevirt kubevirt-config \
 This ConfigMap needs to be created before deployment or the
 virt-controller deployment has to be restarted.
 
-Hugepages support
-^^^^^^^^^^^^^^^^^
-
-For hugepages support you need at least Kubernetes version `1.9`.
-
-Enable feature-gate
-+++++++++++++++++++
-
-To enable hugepages on Kubernetes, check the
-https://kubernetes.io/docs/tasks/manage-hugepages/scheduling-hugepages/[official
-documentation].
-
-To enable hugepages on OKD, check the
-https://docs.openshift.org/3.9/scaling_performance/managing_hugepages.html#huge-pages-prerequisites[official
-documentation].
-
-Pre-allocate hugepages on a node
-++++++++++++++++++++++++++++++++
-
-To pre-allocate hugepages on boot time, you will need to specify
-hugepages under kernel boot parameters `hugepagesz=2M hugepages=64` and
-restart your machine.
-
-You can find more about hugepages under
-https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt[official
-documentation].
-
-Cluster side add-on deployment
+Deploying KubeVirt on Kubernetes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Core components
-^^^^^^^^^^^^^^^
+KubeVirt can be installed on a Kubernetes cluster using the KubeVirt operator.
 
-Once Kubernetes is deployed, you will need to deploy the KubeVirt
-add-on.
+The example below demonstrats how to install KubeVirt using the operator using
+a specific upstream version of KubeVirt.
 
 The installation uses the KubeVirt operator, which manages lifecycle of all KubeVirt core components:
 
 [source,bash]
 ----
+# Pick an upstream version of KubeVirt to install
+$ export VERSION=v0.24.0
```

Comment:
We should probably bump the guide to 0.26.

Reply:
yup